nibabel/tests/test_volumeutils.py

```diff
@@ -607,7 +607,11 @@ def test_a2f_nanpos():
 
 def test_a2f_bad_scaling():
     # Test that pathological scalers raise an error
-    NUMERICAL_TYPES = sum((sctypes[key] for key in ['int', 'uint', 'float', 'complex']), [])
+    NUMERICAL_TYPES = [
+        x
+        for sublist in (sctypes[key] for key in ('int', 'uint', 'float', 'complex'))
+        for x in sublist
+    ]
```

Comment:
No need for a generator. Might need a rerun of `ruff format`, though...

```suggestion
    NUMERICAL_TYPES = [tp for kind in ('int', 'uint', 'float', 'complex') for tp in sctypes[kind]]
```